EpiAware/test/test_utilities.jl

```diff
@@ -45,16 +45,33 @@ end
         @test_throws AssertionError create_discrete_pmf(dist, Δd = 3.0, D = 1.0)
     end
 
-    # Test case 4: Testing output against expected PMF
+    # Test case 4: Testing output against expected PMF basic version - single
+    # interval censoring
     @testset "Test case 4" begin
         dist = Exponential(1.0)
         expected_pmf = [(exp(-(t - 1)) - exp(-t)) / (1 - exp(-5)) for t = 1:5]
-        pmf = create_discrete_pmf(dist, Δd = 1.0, D = 5.0)
+        pmf = create_discrete_pmf(dist, Val(:basic); Δd = 1.0, D = 5.0)
         @test pmf ≈ expected_pmf atol = 1e-15
     end
 
-end
+    # Test case 5: Testing output against expected PMF basic version - double
+    # interval censoring
+    @testset "Test case 5" begin
+        dist = Exponential(1.0)
+        expected_pmf_uncond = [exp(-1);
+                        [(1 - exp(-1)) * (exp(1) - 1) * exp(-s) for s = 1:9]
```

Comment:
Can you throw a link to where you derive this? Or put it somewhere findable so that its origin doesn't get lost? (I like this test and am planning on re-using it)

Reply:
yes I agree - ideally we would have literate docs containing the above as I think it might be hard for people to follow

Reply:
Its a pretty straightforward calc; its with probability 1 been written down somewhere.

Without a link, I could add some latex to the docstring?

Reply:
I think using [this link](https://github.com/CDCgov/Rt-without-renewal/pull/10#issuecomment-1935759767) is plenty

Reply:
I don't think we want to link out to a PR discussion tbh as if we version it it can be improved in the future. Can it just be included as a markdown help file and linked to that way?

Reply:
I've added the analytical derivation to the predictive checking for this method and checked that it renders to nice markdown with `Literate`, which means it'll eventually be part of the docs without further modification [here](https://github.com/CDCgov/Rt-without-renewal/blob/147e97d2e883ad86b9ec4128b47ae84da9335913/EpiAware/test/predictive_checking/discretized_pmfs.jl#L11).